docs/generating-code-from-schema.md

```diff
@@ -230,30 +230,140 @@ Finally, `maxProjectionDepth` will instruct codegen to stop generating beyond 2
-| includeMutations | Generate Query API only for the given list of Mutation fields | All mutations defined in schema |
-| includeSubscriptions | Generate Query API only for the given list of Subscription fields | All subscriptions defined in schema |
-| skipEntityQueries | Disable generating Entity queries for federated types | false |
-| shortProjectionNames | Shorten class names of projection types. These types are not visible to the developer. | false |
-| maxProjectionDepth | Maximum projection depth to generate. Useful for (federated) schemas with very deep nesting | 10 |
+| generatedSourcesDir | Build directory for Gradle                                                                                                                                                                  | build |
+| outputDir | Sub directory of the `generatedSourcesDir` to generate into                                                                                                                                 | generated |
+| exampleOutputDir | Directory to generate datafetcher example code to                                                                                                                                           | generated-examples |
+| includeQueries | Generate Query API only for the given list of Query fields                                                                                                                                  | All queries defined in schema |
+| includeMutations | Generate Query API only for the given list of Mutation fields                                                                                                                               | All mutations defined in schema |
+| includeSubscriptions | Generate Query API only for the given list of Subscription fields                                                                                                                           | All subscriptions defined in schema |
+| skipEntityQueries | Disable generating Entity queries for federated types                                                                                                                                       | false |
+| shortProjectionNames | Shorten class names of projection types. These types are not visible to the developer.                                                                                                      | false |
+| maxProjectionDepth | Maximum projection depth to generate. Useful for (federated) schemas with very deep nesting                                                                                                 | 10 |
+| includeImports             | Maps the custom annotation type to the package, the annotations belong to. Only used when generateCustomAnnotaions is enabled.                                                              |                                                       |
```

Comment:
Please specify the default values.

Reply:
@srinivasankavitha  There are no default values for includeImports and IncludeEnumImports since the packaging is specific to the user implementation